### PR TITLE
C#: Fix dictionary key lookup documentation

### PR DESCRIPTION
**Pull request by raulsntos**
The method to check if a key exists in the dictionary is called `ContainsKey`.

### CODE REVIEW

File: doc/classes/EditorImportPlugin.xml [hunk 1]
```diff
@@ -160,9 +160,9 @@
 				public void GetOptionVisibility(string option, Godot.Collections.Dictionary options)
 				{
 				    // Only show the lossy quality setting if the compression mode is set to "Lossy".
-				    if (option == "compress/lossyQuality" &amp;&amp; options.Contains("compress/mode"))
+				    if (option == "compress/lossyQuality" &amp;&amp; options.ContainsKey("compress/mode"))
 				    {
-				        return (int)options["compress/mode"] == COMPRESS_LOSSY; // This is a constant you set
+				        return (int)options["compress/mode"] == CompressLossy; // This is a constant you set
```
[the remark this answers is not in the document]
Comment (RedworkDE) [reply doc/classes/EditorImportPlugin.xml:165]: The example seems to be taken from: https://github.com/godotengine/godot/blob/2e273f0e35800f2bcc7c05db54100caa97928f69/editor/import/resource_importer_layered_texture.cpp#L122-L125

There `COMPRESS_LOSSY` is a member of a local non-exposed enum.

Tbh to simplify this example I would just make this a string option, eg `return options["compress/mode"] == "lossy";` and get rid of the comment and the weird implied declaration.

Also `lossyQuality` should probably stay snake case as it is a string.

[the remark this answers is not in the document]
Comment (raulsntos) [reply doc/classes/EditorImportPlugin.xml:165]: > I would just make this a string option, eg `return options["compress/mode"] == "lossy";`

But the option is likely stored as an int, so comparing against a string would return `false`.

> Also `lossyQuality` should probably stay snake case as it is a string.

Oh yeah, I totally missed this.

[the remark this answers is not in the document]
Comment (RedworkDE) [reply doc/classes/EditorImportPlugin.xml:165]: My idea was, while the example is copied from some engine code, it is an entirely standalone example in the docs, there is no bits on the page where these options are used elsewhere, so we can just change what the code does exactly to simplify it.

Also while double checking this, I noticed that the bigger example at the top of the documentation also has the camel case string issue in the `_GetImportOptions` method, and there at least the defaultValue one is an definite error, so would be good if you could maybe just fix that one as well.

**Review by mhilbrunner — APPROVED** [review]

File: doc/classes/EditorImportPlugin.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (raulsntos) [reply doc/classes/EditorImportPlugin.xml:165]: > we can just change what the code does exactly to simplify it.

Yes, we probably could but I'd rather let someone else do it as I'm not very familiar with this API.

> Also while double checking this, I noticed that the bigger example at the top of the documentation also has the camel case string issue in the `_GetImportOptions` method

Thanks, fixed.

**Review by RedworkDE — APPROVED** [review]

**Comment by YuriSizov** [comment]
Thanks!

**Comment by YuriSizov** [comment]
Cherry-picked for 4.0.4.

**Pull request merged**